packages/sync/src/offline/OfflineRestoreHandler.ts

```diff
@@ -2,6 +2,7 @@ import ApolloClient from "apollo-client";
 import { NormalizedCacheObject } from "apollo-cache-inmemory";
 import { PersistentStore, PersistedData } from "../PersistentStore";
 import { OperationQueueEntry } from "../links/OfflineQueueLink";
+import {NetworkStatus} from "./NetworkStatus";
 
 /**
  * Class used to restore offline queue after page/application restarts.
```

Comment:
See comment here. This is not actually build to succeed it is to restore the links. So if we are offline actual queue link will be closed and it will hold off those requests till will become online.